_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -10,30 +10,39 @@ author: sibelius
 >
 > -- Orta
 
-Data fetching is a hard problem. How to ask data to a server? When to ask data to server? How to make sure you have
-all necessary data to render your View? How to handle lazy loading? When to lazy load data? When to prefetch data?
+Data fetching is a hard problem: How to ask for data from a server? When is the right time to request data? How know
+you have all necessary data to render your views? Can you do lazy loading? When should you trigger lazy loading of
+data? What about to pre-fetching data?
 
-Relay Modern is a framework to build data driven applications that solves data fetching problems. For introduction
-to Relay Modern reads their docs, and check my Relay Modern talk in [React Conf BR][rbr]
+[Relay][relay] is a framework for building data-driven applications which handles data fetching for you. For an
+introduction to Relay, read [their docs][relay], and also check out my Relay talk at [React Conf BR][rbr].
 
 > You don’t deep dive if you don’t know how to swim
 
 ## TL;DR Relay Modern Network
 
-Relay will aggregate all components data requirements (fragments) and create a request to fulfill it. The network is
-responsible to get this request, send it to a server or a local graphql and return the response data to it.
+Relay will aggregate the data requirements (fragments) for your components, then create a request to fulfill it. The
+API to do this is via the [Relay Environment][env]:
```

Comment:
I wanted to explain why you even have a network function, and how it fits into the relay provided API